storm-client/src/jvm/org/apache/storm/Config.java

```diff
@@ -1481,6 +1481,12 @@
     @isPositiveNumber
     public static final String NUM_STAT_BUCKETS = "num.stat.buckets";
 
+    /**
+     * The path to KafkaClient's jaas.conf when use storm-kakfa-client in Security mode.
+     */
+    @isString
+    public static final String KAFKASPOUT_JAASCONF_PATH = "kafkaspout.jaasconf.path";
```

Comment:
Similar to the above, if possible I think this would fit better in KafkaSpoutConfig.